src/Symfony/Component/Security/Core/Security.php

```diff
@@ -69,4 +70,15 @@ public function getToken(): ?TokenInterface
     {
         return $this->container->get('security.token_storage')->getToken();
     }
+
+    /**
+     * Logout the current user automatically. Dispatch the logout event.
+     */
+    public function autoLogout(): void
```

Comment:
I'm not sure if `auto` is the best wording here, I suggested programmaticLogout in #40663 so I am biased perhaps, but it sounds to me like this isn't automatic if one has to call it :)

Not sure what other options there may be that make this clear enough to everyone.

Reply:
What about just `logout()`?

Reply:
Method name change to `logout()`.